From david-b@pacbell.net  Thu Jul  3 16:17:21 2008
From: David Brownell <david-b@pacbell.net>
Date: Tue, 1 Jul 2008 13:14:17 -0700
Subject: usb gadget: minor USBCV fix for composite framework
To: Greg KH <greg@kroah.com>
Cc: linux-usb@vger.kernel.org, David Lopo <dlopo@chipidea.mips.com>
Message-ID: <200807011314.17401.david-b@pacbell.net>
Content-Disposition: inline


From: David Lopo <dlopo@chipidea.mips.com>

Fill in a reserved/unused device qualifier field to ensure that
the USBCV tests will always pass.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/composite.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/composite.c
+++ b/drivers/usb/gadget/composite.c
@@ -294,6 +294,7 @@ static void device_qual(struct usb_compo
 	/* ASSUME same EP0 fifo size at both speeds */
 	qual->bMaxPacketSize0 = cdev->desc.bMaxPacketSize0;
 	qual->bNumConfigurations = count_configs(cdev, USB_DT_DEVICE_QUALIFIER);
+	qual->bRESERVED = 0;
 }
 
 /*-------------------------------------------------------------------------*/
